strictequal.js

```diff
@@ -0,0 +1,11 @@
+
+const strictEqual = (1, 1) => {
+    var type = Object.prototype.toString.call(value);
```

Comment:
If there are red squiggly lines on your code that means there is something broken or that is not right.  That is your cue to fix that.  Your function is not written properly and so the code after the break is unreachable and not being exported to your test file.